configure.ts

```diff
@@ -31,14 +31,36 @@ export async function configure(command: ConfigureCommand) {
   /**
    * Define environment variables
    */
-  await codemods.defineEnvVariables({ KAFKA_ENABLED: true })
+  await codemods.defineEnvVariables({
+    KAFKA_BROKERS: 'localhost:9092',
+  })
+
+  const project = await codemods.getTsMorphProject()
+  if (project) {
+    const envValidationsFile = await project.getSourceFileOrThrow(command.app.startPath('env.ts'))
+    const kafkaEnvModule = `${command.name}/env`
+
+    if (!envValidationsFile.getImportDeclaration(kafkaEnvModule)) {
+      envValidationsFile.addImportDeclaration({
+        namedImports: ['KafkaEnv'],
+        moduleSpecifier: kafkaEnvModule,
+      })
+    }
+
+    await envValidationsFile.emit()
+  }
```

Comment:
This adds the `KafkaEnv` named import such that in `defineEnvValidations` we can use `KafkaEnv.schema.brokers()`